tock/api/renderers.py

```diff
@@ -16,3 +21,38 @@ def render(self, data, media_type=None, renderer_context=None):
         if not self.headers and len(data) > 0:
             self.headers = data[0].keys()
         return super(PaginatedCSVRenderer, self).render(data, media_type, renderer_context)
+
+def stream_csv(queryset, serializer):
+    """
+    Stream data as CSV, given an interable queryset and a DRF
+    Serializer instance with a .fields dict and .to_representation()
+    method.
+    """
+    rows = map(serializer.to_representation, queryset)
+    fields = list(serializer.fields.keys())
+    response = StreamingHttpResponse(generate_csv(rows, fields), content_type='text/csv')
```

Comment:
No need to assign `response` to a variable--we can just `return StreamingHttpResponse(...)`.